operators/monitoring/README.md

```diff
@@ -0,0 +1,5 @@
+# kube-prometheus-stack for monitoring
+
+<https://github.com/prometheus-operator/kube-prometheus>
+
+Uses namespace: `monitoring`
```

Comment:
So I know we've been sticking READMEs here but it might be better to start putting them in the docs and symlinking?

Reply:
Made a docs page 👍